rotkehlchen/tests/unit/test_blockchain_balances.py

```diff
@@ -264,7 +264,10 @@ def mock_default_call_order(skip_etherscan: bool = False):  # pylint: disable=un
                 amount=FVal('8.204435619126641457'),
                 usd_value=FVal('12.3066534286899621855'),
             ),
-            usdc: Balance(amount=FVal('0.33078')),
-            weth: Balance(amount=FVal('0.007712106620416874')),
-            usdt: Balance(amount=FVal('0.074222')),
+            usdc: Balance(amount=FVal('0.33078'), usd_value=FVal(0.496170)),
```

Comment:
why did the test change also? Due to the token price query reversal?

Reply:
yes, it changed from zero to using the actual price